Better processing of unconfirmed transactions

Senyoret1 opened the pull request:
Fixes #280 

Changes:
- The transaction details page does not show the row with the block number if the transaction is not confirmed.
- The address details page now shows separately the confirmed and unconfirmed balances and totals:
![a](https://user-images.githubusercontent.com/34079003/53192003-abf02080-35e3-11e9-8e33-edb6397155ea.png)

Senyoret1 commented:
@gz-c Maybe "pending" could cause confusion. In "Total Received" the word "pending" makes sense because that row first shows the coins in the confirmed txs and then, as pending, the coins in the unconfirmed txs, so after confirming all the txt the total will be the sum of both values. However, in "Current Coins" the situation is a bit different, because the first value is how many coins the address has now and the second one is how many it will have after confirming all the txs, so there's no need to sum the values.

I think that the two alternatives would be:
- Use "pending" in "Current Coins" and "Current Hours", but instead of showing the predicted value returned by the API, show `predicted - current`.
- Replace "predicted" with "after confirming the pending transactions" or something else.

mahansky commented:
I would go with the first option, also show "+" or "-" sign so something like 
`Current Hours: 70   (+30 pending)`

There is also a little mistake that says "SKY" next to the predicted hours.

Senyoret1 commented:
I modified the code using the first option and applied the suggestions made by @mahansky . Additionally, I deleted the "SKY" text from all the pending values, since after removing it from the pending hours it looked a bit strange having it in some places and not in others:
![pending](https://user-images.githubusercontent.com/34079003/54556214-c58c4a00-498e-11e9-96f8-20a512b75c8f.png)
